<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>얼굴인식 로그인</string>
  </property>
  <widget class="QLabel" name="cam">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cam_on">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>700</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;HY엽서L&quot;;</string>
   </property>
   <property name="text">
    <string>얼굴인식 시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_main">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>1100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
